src/index.ts

```diff
@@ -4,6 +4,7 @@ import minimist from 'minimist';
 import {AndroidSetup} from './commands/android';
 import {IosSetup} from './commands/ios';
 import {AVAILABLE_COMMANDS} from './constants';
+import {AndroidSubcommand} from './commands/android/subcommands/index';
```

Comment:
No need to mention `/index`.

```suggestion
import {AndroidSubcommand} from './commands/android/subcommands';
```